examples/with-fauna/components/EntryForm.js

```diff
@@ -0,0 +1,60 @@
+'use client'
+
+import cn from 'classnames'
+import { createEntryAction } from '@/actions/entry'
+import { experimental_useFormState as useFormState } from 'react-dom'
```

Comment:
Heads up: you'll want a `// @ts-ignore` here for right now, as you move to TS. Known and being fixed